medium/profiles/migrations/0001_initial.py

```diff
@@ -19,19 +19,19 @@ class Migration(migrations.Migration):
             name='Profile',
             fields=[
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('about_text', models.TextField()),
+                ('profile_pic', models.ImageField(upload_to=medium.profiles.utils.get_profile_pic_url)),
```

Comment:
stick to the naming scheme if there is any field name doesn't make sense
make sure to change the field name in schema first then reflect changes in the implementation.